src/runtime/data/kata-collect-data.sh

```diff
@@ -5,8 +5,14 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+PROJECT_BUG_URL="https://github.com/kata-containers/kata-containers/issues/new"
+
+RUNTIME_VERSION_INFO=$( kata-runtime --version | sed -n 's/^.*: //p' )
+read -r VERSION COMMIT <<< $( echo $RUNTIME_VERSION_INFO )
+PROJECT_NAME="Kata Containers"
```

Comment:
`PROJECT_NAME` should be in lower case (not a environment variable) and typeset'ed alongside the others.

Reply:
changed